src/Examples/DocScan/DocScanExample/Controllers/DbsController.cs

```diff
@@ -55,15 +50,15 @@ public IActionResult Index()
                     .Build()
                     )
                 .WithCreateIdentityProfilePreview(true)
-                 .WithIdentityProfileRequirements(new
+                 /*.WithIdentityProfileRequirements(new
```

Comment:
Uncomment the Identity Profile Requirements.

Reply:
Uncommented to restore dbs status